docs/user/document/category-usage.en.md

```diff
@@ -1,5 +1,24 @@
 # Category - Usage
 
-* For a document with the profile `CSAF Base` the category value must not be equal to the (case insensitive) name (without the prefix `csaf_`) or value of any other profile than "CSAF Base".
-  Any occurrences of dash, whitespace, and underscore characters are removed from the values on both sides before the match.
-  Also, the value must not start with the reserved prefix `csaf_` except if the value is `csaf_base`.
+This value sets and should always be the profile name for the CSAF document.
```

Comment:
```suggestion
This value determines the profile name for the CSAF document. It should always be the value of the profile defined in the [specification]().
```
Potentially, link section 4 of the spec at [...](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html#4-profiles).

Reply:
like this?